pages/support.html

```diff
@@ -14,42 +14,51 @@ <H1>{% trans "Community Support" %}</H1>
   <div style="clear: both;"></div>
 
   <div class="halfbox_left">
-    <H2>{% trans "Wiki" %}</H2>
+    <h2>{% trans "Community Forums" %}</h2>
     <p>
-      {% blocktrans with mixxx_wiki_link="/wiki/" beginner_dj_links="/wiki/doku.php/beginner_dj_links" wiki_troubleshooting="/wiki/doku.php/troubleshooting" %}The <a href="{{ mixxx_wiki_link }}">Mixxx Wiki</a> has a wealth of useful documentation for users and developers. New to it all and not sure how to get started? Checkout the <a href="{{ beginner_dj_links }}">Beginner DJ Links</a>. Running into technical issues? See the <a href="{{ wiki_troubleshooting }}">Troubleshooting</a> guide. {% endblocktrans %}
+      {% blocktrans with forums_link="/forums" %}Have a Mixxx-related question? Something other DJs can help you with? Ask in the <a href="{{ forums_link }}">Mixxx Community Forums</a>.{% endblocktrans %}
     </p>
   </div>
 
   <div class="halfbox_right">
-    <H2>{% trans "User Manual" %}</H2>
+    <h2>{% trans "Wiki" %}</h2>
     <p>
-      {% blocktrans with manual_latest_link="/manual/latest" manual_beginner_link="/manual/latest/chapters/djing_with_mixxx.html" %}The <a href="{{ manual_latest_link }}">Mixxx Manual</a> provides an overview of Mixxx's user interface, information about <b>configuration</b> and hardware, and much more. It also contains useful information for <b>beginners</b> under <a href="{{ manual_beginner_link }}">DJing with Mixxx</a>.{% endblocktrans %}
+      {% blocktrans with mixxx_wiki_link="/wiki/" beginner_dj_links="/wiki/doku.php/beginner_dj_links" wiki_troubleshooting="/wiki/doku.php/troubleshooting" %}The <a href="{{ mixxx_wiki_link }}">Mixxx Wiki</a> has a wealth of useful documentation for users and developers. New to it all and not sure how to get started? Checkout the <a href="{{ beginner_dj_links }}">Beginner DJ Links</a>. Running into technical issues? See the <a href="{{ wiki_troubleshooting }}">Troubleshooting</a> guide. {% endblocktrans %}
     </p>
   </div>
+ 
   <div style="clear: both;"></div>
-
+  
   <div class="halfbox_left">
-    <h2>{% trans "Community Forums" %}</h2>
+    <h2>{% trans "User Manual" %}</h2>
     <p>
-      {% blocktrans with forums_link="/forums" %}Have a Mixxx-related question? Something other DJs can help you with? Ask in the <a href="{{ forums_link }}">Mixxx Community Forums</a>.{% endblocktrans %}
+      {% blocktrans with manual_latest_link="/manual/latest" manual_beginner_link="/manual/latest/chapters/djing_with_mixxx.html" %}The <a href="{{ manual_latest_link }}">Mixxx Manual</a> provides an overview of Mixxx's user interface, information about <b>configuration</b> and hardware, and much more. It also contains useful information for <b>beginners</b> under <a href="{{ manual_beginner_link }}">DJing with Mixxx</a>.{% endblocktrans %}
```

Comment:
Why is "configuration" and "beginners" in bold ? these are the only two words on this page that are bold and I think they don't deserve it.